app/javascript/angular/code/controllers/info_window_ctrl.js

```diff
@@ -1,6 +1,8 @@
-function InfoWindowCtrl($scope, sensors, infoWindow ) {
-  $scope.sensors = sensors;
-  $scope.infoWindow = infoWindow;
```

Comment:
Here we were putting `infoWIndow` on the `$scope`. I see the new code does not do that. Was that dead code?

Reply:
No! Good catch, thanks!